src/app/home/home.component.html

```diff
@@ -3,36 +3,36 @@
         <img src="../../assets/CashmereLogo.svg" height="190px" width="auto" alt="Fabric.Cashmere Logo"/>
     </span>
     <span class="headerRight">
-        <img src="../../assets/FabricCashmere.svg" height="55px" width="auto" alt="Fabric.Cashmere Title"/>
+        <img src="../../assets/FabricCashmere.svg" height="100%" width="auto" alt="Fabric.Cashmere Title"/>
         <p class="headerCopy">Health Catalyst's living style guide for Angular</p>
         <button hc-button title="Primary Button" color="primary" routerLink="/demo">Get Started</button>
     </span>
 </header>
 <main>
     <section>
-        <span><img src="../../assets/CashmereHome_UX.png" height="370px" width="auto" alt="Cashmere UX Illustration"/></span>
-        <span class="sectionCenter"  style="height: 370px;">
-            <div style="padding-left: 100px;">
+        <span class="sectionOneImg"><img src="../../assets/CashmereHome_UX.png" alt="Cashmere UX Illustration"/></span>
+        <span class="sectionCenter"  style="max-height: 370px;">
+            <div class="sectionOneText">
```

Comment:
Thanks for taking out these inline styles

Reply:
Thanks Cory, and agreed on the NavBar, I was thinking the same thing.  Also it will be a good enticement for developers to use our navbar over their's if it offers things like built in responsiveness.  I'll add an issue for this and I volunteer myself to get it started.